bin/codemods/run.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	const excludeUtilityFiles = name => name !== 'config.js' && name !== 'helpers.js';
+	const excludeDotFiles = name => name[0] !== '.';
+	const includeJSFiles = name => path.extname( name ) === '.js';
+	return fs.readdirSync( './bin/codemods/src' ).filter(
+		excludeUtilityFiles
+	).filter(
+		excludeDotFiles
+	).filter(
+		includeJSFiles
+	).map( name => name.replace( '.js', '' ) );
+
+}
+
+function getValidCodemodNames() {
+	return [
+		...getLocalCodemodFileNames(),
+		...Object.getOwnPropertyNames( config.codemodArgs ),
+	].map( name => '- ' + name ).sort();
+}
+
+function printableValidCodemodNames() {
+	return `${ getValidCodemodNames().join('\n') }`
+}
+
+function generateBinArgs( name ) {
+	if ( config.codemodArgs.hasOwnProperty( name ) ) {
+		// Is the codemod defined in the codemodArgs object?
+		return config.codemodArgs[ name ];
+	} else if ( getLocalCodemodFileNames().includes( name ) ) {
+		// Is the codemod a local script defined in bin/codemods/src folder?
+		return [
+			`--transform=bin/codemods/src/${ name }.js`,
+		];
+	} else {
+		throw new Error(
+			`"${ name }" is an unrecognized codemod.`
+		);
+	}
+}
+
+function runCodemod() {
+	const args = process.argv.slice( 2 );
+	if ( args.length === 0 || args.length === 1 ) {
+		process.stdout.write( '\n' );
+		process.stdout.write( './bin/codemods/run.js [transformation name] [target(s)]\n' );
+		process.stdout.write( '\n' );
+		process.stdout.write( `Valid transformation names: \n${ printableValidCodemodNames() }\n` );
+		process.stdout.write( '\n' );
+		process.stdout.write( 'Example: "./bin/codemods/run.js commonjs-imports client/blocks client/devdocs"\n' );
+		process.stdout.write( '\n' );
+		process.exit( 0 );
```

Comment:
not sure if it would help or not, but we could also create an array first then print…

```js
process.stdout.write( [
	'',
	'./bin/codemods/run.js …',
	'',
	'Valid transformation names:',
	getValidCodemodNames().join( '\n' ),
	'',
	'Example: "…"',
	'',
].join( '\n' ) )
```